docs/settings/logging-settings.asciidoc

```diff
@@ -33,7 +47,7 @@ logging:
 ----
 
 [[log-in-json-ECS-example]]
-==== Log in json format
+===== Log in json format
```

Comment:
Also check other uses of json in this section to see if they should be JSON

```suggestion
===== Log in JSON format
```

Reply:
Applied the changes to the places where it's text (but not in code) :)